Tools/AP_Bootloader/AP_Bootloader.cpp

```diff
@@ -134,6 +134,21 @@ int main(void)
         try_boot = false;
         led_set(LED_BAD_FW);
     }
+#if AP_BOOTLOADER_NETWORK_ENABLED
+    if (ok == check_fw_result_t::CHECK_FW_OK) {
```

Comment:
i'd prefer the memmem here or via a helper function, and not have the global variable

Reply:
done